gc/base/MemorySubSpace.cpp

```diff
@@ -919,16 +919,23 @@ MM_MemorySubSpace::systemGarbageCollect(MM_EnvironmentBase* env, uint32_t gcCode
 	}
 
 	if (_collector && _usesGlobalCollector) {
-		/* TODO: This is bogus for multiple memory spaces - should ask the space, not the heap */
-		_extensions->heap->getResizeStats()->setFreeBytesAtSystemGCStart(getApproximateActiveFreeMemorySize());
-
 		env->acquireExclusiveVMAccessForGC(_collector);
-		reportSystemGCStart(env, gcCode);
-
-		/* system GCs are accounted into "user" time in GC/total time ratio calculation */
-		_collector->garbageCollect(env, this, NULL, gcCode, NULL, NULL, NULL);
-
-		reportSystemGCEnd(env);
+		#if defined(OMR_GC_IDLE_HEAP_MANAGER)
+		if (isSystemGCAllowed(env, gcCode)) {
+		#endif
+			/* TODO: This is bogus for multiple memory spaces - should ask the space, not the heap */
+			_extensions->heap->getResizeStats()->setFreeBytesAtSystemGCStart(getApproximateActiveFreeMemorySize());
```

Comment:
Since we are supposedly idling, exclusive access acquire should be farily cheap to obtain, so we not saving that much by not doing that 'exercise' after all. 

If if end up just acquiring pool lock, i'll add that 
1) the thread must have vm access as a min (to prevent any other party acquiring exclusive and messing up with heap (like very GC just from another thread).
2) i'd like to see verbose GC stanza reporting this decommit if running standalone (probalby even if it's run within sysGC, although that would be implied from sysGC reason'). But I want to see durationms of the operation, in any case. This would not be really a gc-op, but more like info (see for example classunload-info stanza).